ui/raidboss/popup-text.js

```diff
@@ -355,7 +363,10 @@ class PopupText {
           let holder = that.alarmText.getElementsByClassName('holder')[0];
           let div = makeTextElement(text, 'alarm-text');
           addText.bind(that)(holder, div);
-          window.setTimeout(removeText.bind(that, holder, div), duration * 1000);
+          window.setTimeout(
+              removeText.bind(that, holder, div),
+              (duration.fromTrigger || duration.alarmText) * 1000
```

Comment:
Can this be max of duration.alarmText and duration.fromTrigger? If somebody specifies a really long alarmText option, a specified duration on the trigger should not make it shorter, I think?

Reply:
Well my idea was as no durationSecond is specified by default, the user had to add it manually therefore he wants it to be the time for this specific trigger.